### ISSUE TITLE
Some emojis are not working

### ISSUE THREAD

**Issue by liza-mae** [buildkite/emojis#385]
Hi,

I am trying to use some emojis listed here, but they are not working and show up blank as follows: 

![Screenshot from 2022-06-24 12-45-57](https://user-images.githubusercontent.com/23424008/175646592-cbffd670-06bd-46cd-8b57-7bbe99172a7e.png)

Some of I have tried that don't work are: large_orange_circle, large_yellow_circle, large_green_circle, large_purple_circle, large_brown_circle

These work though: white_circle, black_circle, red_circle, large_blue_circle

Can someone help?

**Comment by yob** [comment]
@liza-mae thanks for the report!

Can I confirm where you are trying to use the emojis that don't work? On buildkite.com? Or (based on the linked issue https://github.com/buildkite/emojis/issues/386) maybe the bk cli?

**Comment by liza-mae** [comment]
@yob I was trying to use the emojis on buildkite.com

**Comment by yob** [comment]
oh, that's not great :thinking: 

To test it, I updated the label on a test step to the following: `large_orange_circle :large_orange_circle: red_circle :red_circle: buildkite :buildkite:`

It renders like this for me on buildkite.com:

![Screenshot from 2022-06-28 21-55-22](https://user-images.githubusercontent.com/8132/176172573-88158378-a0a7-4f9b-beee-b610da5234d7.png)

When I inspect the DOM, it looks like this:

![Screenshot from 2022-06-28 21-56-57](https://user-images.githubusercontent.com/8132/176172733-151ba91b-240d-412c-a8d3-177ae64ef48b.png)

This repo defines a mixture of official unicode emojis and custom "emojis" (logos, etc). When rendered on buildkite.com, we use utf8 encoded data for the official unicode emojis, and an img tag for the custom "emojis". Is it possible your system has no fonts with large_orange_circle, large_yellow_circle, large_green_circle, large_purple_circle, or large_brown_circle?

Picking two emojis from your lists at random, [it looks like red circle was released in 2010](https://emojipedia.org/large-red-circle/), and [large_orange_circle was released in 2019](https://emojipedia.org/large-orange-circle/).
